ci/d2-convert-tag.sh

```diff
@@ -29,5 +24,5 @@ then
 	set +x
 	git -c "credential.https://github.com.username=${OCEAN_D2_USER}" \
 		-c "core.askPass=./ci/askpass.sh" \
-		push "https://github.com/sociomantic-tsunami/ocean-d2.git" "$d2tag"
+		push "https://github.com/sociomantic-tsunami/ocean.git" "$d2tag"
```

Comment:
Oh, this is also wrong, I think, because we need to add the `+d2` suffix so they don't oeverride the original tag.

Also, for this step I think the [deploy](https://docs.travis-ci.com/user/deployment/) section is more appropriate, so we can use the condition (`on: tag: true`) to only run when is a tag (and also to "error" the build if it fails, `after_success` exit code is ignored).

> The exit code of `after_success`, `after_failure`, `after_script` and subsequent stages do not affect the build result. However, if one of these stages times out, the build is marked as a failure.